### ISSUE TITLE
Ontology: add concept of 'dimension' of an entity or enrichment

### ISSUE THREAD

**Issue by lyndonnixon** [linkedtv/wp2#18]
Jaap, 12.8.: "the concept of dimension/intension should be added to the LinkedTV model. It gives the user the flexibility to express the intention for using enrichments + entities"

**Comment by jblom** [comment]
It could be expressed as an additional oa:motivatedBy attached to an annotation? For this we could choose from: oa:bookmarking, oa:classifying, oa:commenting, oa:describing, oa:editing, oa:highlighting, oa:identifying, oa:linking, oa:moderating, oa:questioning, oa:replying, oa:tagging

I'd say when dimensions could be things like: "background information", "opinions", etc oa:tagging seems the most appropriate? (see http://www.w3.org/ns/oa#d4e731)

**Comment by rtroncy** [comment]
First, if this is a bug related to the LinkedTV ontologies, it should be reported at https://github.com/linkedtv/ontologies.

Per [definition](http://www.w3.org/ns/oa#d4e555), each Annotation SHOULD have at least one oa:motivatedBy relationship to an instance of oa:Motivation, which is a subClass of skos:Concept. A number of pre-existing motivations have been registered in the [oa:motivationScheme](motivationScheme). So far, we adopted the generic oa:linking motivation for ALL our enrichments. One possibility could be to create new skos:Concept in an extended motivationScheme, adapted to LinkedTV.

Closing and marking as duplicate of https://github.com/linkedtv/ontologies/issues/7
